specification/keyvault/data-plane/readme.python.md

```diff
@@ -4,65 +4,65 @@ These settings apply only when `--python` is specified on the command line.
 Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
 Use `--python-mode=update` if you already have a setup.py and just want to update the code itself.
 
-``` yaml $(python)
-python-mode: create
-python:
-  azure-arm: true
-  license-header: MICROSOFT_MIT_NO_VERSION
-  payload-flattening-threshold: 2
-  package-name: azure-keyvault
-  clear-output-folder: true
-  no-namespace-folders: true
+``` yaml
```

Comment:
I'd say in theory it's safe to drop the Python scope but I wonder if there's a case where it's better to have it for the sake of safety. Also the `$(python)` check is probably good to keep around, but obviously not necessary if AutoRest only loads this file when --python is specified in the command line.

Was there something about multiapi that made this necessary, perhaps a bug in AutoRest that needs to be fixed?

Reply:
I don't think there's any issue with the current autorest, I just got rid of it because I'm used to getting rid of it in READMEs. It should work with the new autorest, but going to test adding back ini the python links and regenerating tomorrow to make sure. thanks!

Reply:
Maybe @lmazuel knows more about the history behind using the language specific scopes and guards in these files.  Perhaps it's not necessary to keep them?  I guess I'm just superstitious when it comes to AutoRest configuration ;)

Reply:
coming back after a long time. they don't matter, mgmt team has some back-compat issues with it, but since this is data plane, we're good to get rid of the flag, since using `--python` will bring you the python.md file